src/main/java/se/liu/ida/hefquin/engine/queryplan/logical/impl/LogicalOpBGPAdd.java

```diff
@@ -45,4 +48,18 @@ public void visit( final LogicalPlanVisitor visitor ) {
 		visitor.visit(this);
 	}
 
+	@Override
+	public String toString(){
+		final StringBuilder builder = new StringBuilder();
+
+		builder.append("> bgpAdd ");
+		builder.append("( ");
+		builder.append( bgp.getTriplePatterns() );
```

Comment:
Does this work as expected? I mean, `getTriplePatterns` returns a `Set`. What does the `toString()` method of `Set` do?